recipes/nspr/all/conanfile.py

```diff
@@ -31,35 +34,38 @@ class NsprConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
         else:
             del self.options.win32_target
 
     def configure(self):
-        del self.settings.compiler.cppstd
-        del self.settings.compiler.libcxx
         if self.options.shared:
             del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = os.path.join("nspr-" + self.version, "nspr")
-        os.rename(extracted_dir, self._source_subfolder)
-        tools.rmdir(os.path.join("nspr-" + self.version))
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination="tmp", strip_root=True)
```

Comment:
Feature request? `strip_root=2`

Reply:
It worths an issue

Reply:
please fill a feature request to conan client